x-pack/plugins/infra/public/lib/lib.ts

```diff
@@ -101,6 +101,7 @@ export type InfraWaffleData = InfraWaffleMapGroup[];
 
 export interface InfraWaffleMapNode {
   id: string;
```

Comment:
This might be a good time to qualify what this `id` is to differentiate it from the newly added `nodeId`. Is it a `pathId`?

Or maybe this indicates that the term `node` for a box in the waffle map conflicts with the use of `node` as a data model entity. We could call it `InfraWaffleMapCell` or `InfraWaffleMapBox`. But it might be something to split out into a separate PR. What do you think?

Reply:
I like `pathId`